lamp_virtual_host

```diff
@@ -64,16 +69,33 @@ check_required_package() {
   fi
 }
 #############################################################
+# Check OS
+#############################################################
+if [ -f /etc/redhat-release ]; then
+  printf "RUN Update\n"
+  sudo yum update
+  
+fi
+
+if [ -f /etc/lsb-release ]; then
+	printf "RUN Update\n"
+	sudo apt-get update && sudo apt-get upgrade
+fi
+#############################################################
 # Check Dependencies
 #############################################################
 # Check Apache
 if [ ! -d /etc/apache2 ]; then
   printf "[${red}${icon_ko}${nocolor}] Sorry, but ${green}/etc/apache2${nocolor} is required.\n"
+  printf "Install Apache2"
+  sudo apt-get install apache2
```

Comment:
non ha senso installare qui apache perché comunque subito dopo lo script esce con exit 1. Quindi si dovrebbe togliere la riga 89 e 92.

Reply:
si potrebbe anche aggiungere la repo di apache in modo da avere sempre la versione più recente